src/test/java/org/ice4j/ice/harvest/SinglePortUdpHarvesterTest.java

```diff
@@ -0,0 +1,96 @@
+package org.ice4j.ice.harvest;
+
+import org.ice4j.Transport;
+import org.ice4j.TransportAddress;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.net.BindException;
```

Comment:
Thank you for adding tests! 

Nits: license header, * imports

Reply:
Le sigh (pardon my French).